src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationUseStyleCheck.java

```diff
@@ -126,48 +126,107 @@
  * Example:
  * </p>
  * <pre>
+ * &#64;SuppressWarnings("unchecked") // OK
  * &#64;Deprecated // OK
- * &#64;SomeArrays(pooches={DOGS.LEO}) // Violation - COMPACT_NO_ARRAY
- * &#64;SuppressWarnings({""}) // Violation - COMPACT_NO_ARRAY
+ * &#64;SomeArrays({"unchecked","unused"}) // OK
  * public class TestOne
  * {
  *
  * }
  *
- * &#64;SomeArrays(pooches={DOGS.LEO}, um={}, test={"bleh"}) // Violation - COMPACT_NO_ARRAY
- * &#64;SuppressWarnings("") // OK
- * &#64;Deprecated() // Violation - cannot have closing parenthesis
+ * &#64;SuppressWarnings(value={"unchecked"}) // Violation - no parameter should be referenced
```

Comment:
`parameter 'value' should not be referenced`

Reply:
Hi, @romani 
For `COMPACT_NO_ARRAY`, I think`no parameter should be referenced` is good. Because any parameters should not be referenced. On the other hand, for `COMPACT`, only `value` parameter should not be referenced. If it is `valu`, there is no problem about that. Hence, using `parameter 'value' should not be referenced` is appropriate. I think it is good to differentiate these two errors since users may confuse about that difference between them. I agree that I should add `parameter` for `’value’ should not be referenced.`
What do you think about this?

Reply:
I just want to point attention o user what is parameter of annotation. No much people know how it is named. Placing one word in front make it clear and user understand what violation mean when no parameter is provided

Reply:
Ok. I got it. I will change it.

Reply:
@romani 
If I put `parameter 'value' should not be referenced`, the number of characters go over 100. Can I move the comment to 2nd line to solve the error? Or is there any other way you suggest to do?

Reply:
can we do `parameter 'value' should not be used` ?

Reply:
@romani 
It does not work at line 199 since it becomes 101 characters. Can I remove a single quotation for value? After I remvoe single quotation, it worked.

Reply:
Use ` shouldn't`

Reply:
@romani 
I updated. Thank you!